app/views/CheckYourAnswersView.scala.html

```diff
@@ -38,7 +39,26 @@
 
      @govukSummaryList(list)
 
-     @h2("checkYourAnswers.submit.h2")
+
+     @itemList.map { case (itemName, item) =>
+        @h2(messages("checkYourAnswers.itemDetails.h2", itemName))
+        @govukSummaryList(item)
+     }
+
+     @if(itemsToAdd) {
+        @link(
+            addItemLink,
+            "checkYourAnswers.add.item.link",
+            id = Some("add-item")
+        )
+     }
+
+    <br>
+    <br>
+    <br>
+    <br>
```

Comment:
Shouldn't use `<br>` for styling the page, you can add a CSS class to add extra margin if we need to space the page out. Maybe inspect the prototype HTML to see how Tom did it.